content/applications/productivity/iot/config/troubleshooting.rst

```diff
@@ -2,133 +2,150 @@
+the server doesn't use a multi-database environment.
 
-If you are not in any of the cases listed above, make sure that the IoT Box
-has correctly started, by checking that a fixed green LED is showing next to
-the micro-USB port.
+To access the database from the :abbr:`IoT (Internet of Things)` box, open a browser and type in the
+database address.
 
-I've connected my IoT Box but it's not showing in my database
--------------------------------------------------------------
+The IoT box is connected to the Odoo database, but cannot be reached
+--------------------------------------------------------------------
 
-When you connect an IoT Box to a database, the IoT Box might restart, if
-that is the case, it might take up to one minute before appearing in your
-database. If after some time the IoT is still not showing, make sure that
-your database can be reached from the IoT Box and that your server doesn't
-use a multi-database environment.
+Make sure that the :abbr:`IoT (Internet of Things)` box and the device (computer) running the
+browser are located on the same network, as the :abbr:`IoT (Internet of Things)` box cannot be
+reached from outside the local network.
 
-My IoT Box is connected to my database, but cannot be reached
--------------------------------------------------------------
+The HTTPS certificate doesn't generate
+--------------------------------------
 
-Make sure that the IoT Box and the device running the browser are located
-on the same network as the IoT Box cannot be reached from outside the local
-network.
+In order to generate a Hypertext Transfer Protocol Secure certificate, an IoT box subscription is
+required for the :abbr:`IoT (Internet of Things)` box. Connecting the :abbr:`IoT (Internet of
+Things)` box prior to configuring an :abbr:`IoT (Internet of Things)` subscription for the database
+and :abbr:`IoT (Internet of Things)` box with the Account Manager will result in an unsecured
+connection.
+
+In addition, a firewall can also prevent the :abbr:`HTTPS (Hypertext Transfer Protocol Secure)`
+certificate from generating correctly. In this case, deactivate the firewall until the certificate
+is successfully generated. It should also be noted that certain devices, such as a router that have
+a firewall built-in to the software of the unit can prevent the :abbr:`HTTPS (Hypertext Transfer
+Protocol Secure)` certificate from generating.
+
+.. seealso::
+   :doc:`HTTPS certificate (IoT) <https_certificate_iot>`
 
 Printer
 =======
 
-My printer is not detected
---------------------------
+The printer is not detected
+---------------------------
 
-If one of your printers doesn't show up in your devices list, go to the
-IoT Box homepage and make sure that it is listed under *Printers*.
+If one of the printers doesn't show up in the devices list, go to the :abbr:`IoT (Internet of
+Things)` box home page and make sure that it is listed under :guilabel:`Printers`.
```

Comment:
The IoT box home page should have a label.

Reply:
Noted, nice catch.